src/extensionFunctions.ts

```diff
@@ -898,3 +903,54 @@ export async function copyFileTextToClipboard(uri: vscode.Uri) : Promise<void> {
 		}
 	}
 } 
+
+export async function pasteClipboardToActiveEditor() : Promise<void> {
+	let currentEditor : vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+	if (!currentEditor) {
+		await vscode.commands.executeCommand('workbench.action.openPreviousRecentlyUsedEditor');
+		currentEditor = vscode.window.activeTextEditor;
+	}
+	if (currentEditor) {
+		const textFromClipboard = await vscode.env.clipboard.readText();
+		if (!textFromClipboard) {
+			await vscode.window.showWarningMessage(`No text found on clipboard`);
+			return;
+		}
+		await currentEditor.insertSnippet(new vscode.SnippetString(textFromClipboard));
+		return;
+	}
+	await vscode.window.showWarningMessage(`No active text editor found to paste from clipboard`);
+}
+
+export async function openEditorForFile(uri: vscode.Uri) : Promise<vscode.TextEditor | undefined> {
```

Comment:
this function is not openign any editors, only providing an opened editor corrseponding to the provided Uri